apps/laboratory/src/components/Wagmi/WagmiConnectButton.tsx

```diff
@@ -83,12 +94,13 @@ export function WagmiConnectButton() {
       <w3m-button />
       {isConnected ? (
         <>
-          <Button onClick={() => onSignMessage()} data-testid="sign-message-button">
+          <Button data-testid="sign-message-button" onClick={() => onSignMessage()}>
             Sign Message
           </Button>
-          <Button onClick={() => onSignTypedData()} data-testid="sign-typed-data-button">
+          <Button data-testid="sign-typed-data-button" onClick={() => onSignTypedData()}>
```

Comment:
```suggestion
          <Button data-testid="sign-typed-data-button" onClick={onSignTypedData}>
```

If no props passed to function, this have a better readability